GNUmakefile

```diff
@@ -306,6 +311,17 @@ clean: lib/clean test/clean headers/clean
 distclean: clean
 	$(RM) make.inc include/lapack/defines.h
 
+# Install git hooks
+hooks = .git/hooks/pre-commit
+
+hooks: ${hooks}
+
+.git/hooks/%: tools/hooks/%
+	@if [ -e .git/hooks ]; then \
```

Comment:
Only one question: why checking if the folder exists and not trying to do `mkdir` ?